src/main/webapp/assets/css/feedStyle.css

```diff
@@ -1,7 +1,4 @@
 #col-container {
-  display: flex;
-  flex-direction: column;
-  flex-wrap: wrap;
   width: 80%;
```

Comment:
I think Areeta added this for making the mobile website look better? Does the mobile still work?

Reply:
mobile still works! the flexbox is around the map-posts container :)

Reply:
Cool!